templates/components/common/share.html

```diff
@@ -61,11 +61,13 @@
             {{/each}}
         </ul>
         <script type="text/javascript"
-                src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4e94ed470ee51e32"></script>
+                defer src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4e94ed470ee51e32"></script>
         <script>
-            if (typeof(addthis) === "object") {
-                addthis.toolbox('.addthis_toolbox');
-            }
+            window.addEventListener('DOMContentLoaded', function() {
```

Comment:
You should use https://api.jquery.com/load-event/ here to ensure that the external script has been downloaded and parsed.